src/plot_api/plot_api.js

```diff
@@ -2092,9 +2092,6 @@ function _relayout(gd, aobj) {
     var oldWidth = gd._fullLayout.width,
         oldHeight = gd._fullLayout.height;
 
-    // coerce the updated layout
-    Plots.supplyDefaults(gd);
```

Comment:
Really? This doesn't break the autosize teats? I thought I tried  removing it at some point a few weeks back. 

Oh well. One less supply defaults loop ia great news 🐎